src/formattedcode/writers.py

```diff
@@ -31,16 +31,20 @@
 import os
 from os.path import abspath
 
+import pluggy
```

Comment:
Could pluggy be encapsulated in plugincode instead rather than being imported here? Or actually I am not sure it is used at all  here and could be removed?